app/components/preprint-status-banner/component.js

```diff
@@ -0,0 +1,99 @@
+import Ember from 'ember';
+
+const PENDING = 'pending';
+const ACCEPTED = 'accepted';
+const REJECTED = 'rejected';
+
+const PRE_MODERATION = 'pre-moderation';
+const POST_MODERATION = 'post-moderation';
+
+export const ICONS = Object.freeze(Ember.Object.create({
+    [PENDING]: 'fa-hourglass-o',
+    [ACCEPTED]: 'fa-check-circle-o',
+    [REJECTED]: 'fa-times-circle-o'
+}));
+
+export const STATUS = Object.freeze(Ember.Object.create({
+    [PENDING]: 'components.preprint-status-banner.pending',
+    [ACCEPTED]: 'components.preprint-status-banner.accepted',
+    [REJECTED]: 'components.preprint-status-banner.rejected'
+}));
+
+export const MESSAGE = Object.freeze(Ember.Object.create({
+    [PRE_MODERATION]: 'components.preprint-status-banner.message.pending_pre',
+    [POST_MODERATION]: 'components.preprint-status-banner.message.pending_post',
+    [ACCEPTED]: 'components.preprint-status-banner.message.accepted',
+    [REJECTED]: 'components.preprint-status-banner.message.rejected'
+}));
+
+export const WORKFLOW = Object.freeze(Ember.Object.create({
+    [PRE_MODERATION]: 'components.preprint-status-banner.pre_moderation',
+    [POST_MODERATION]: 'components.preprint-status-banner.post_moderation'
+}));
+
+export const CLASS_NAMES = Object.freeze(Ember.Object.create({
+    [PRE_MODERATION]: 'preprint-status-pending-pre',
+    [POST_MODERATION]: 'preprint-status-pending-post',
+    [ACCEPTED]: 'preprint-status-accepted:',
+    [REJECTED]: 'preprint-status-rejected'
+}));
+
+
+export default Ember.Component.extend({
+    i18n: Ember.inject.service(),
+    theme: Ember.inject.service(),
+
+    // translations
+    labelModeratorFeedback: 'components.preprint-status-banner.feedback.moderator_feedback',
+    moderator: 'components.preprint-status-banner.feedback.moderator',
+    feedbackBaseMessage: 'components.preprint-status-banner.feedback.base',
+    baseMessage: 'components.preprint-status-banner.message.base',
+
+    classNames: ['preprint-status-component'],
+    classNameBindings: ['getClassName'],
+
+    getClassName: Ember.computed('model.provider.reviewsWorkflow', 'model.reviewsState', function() {
+        if (this.get('model.reviewsState') === PENDING) {
+            return CLASS_NAMES[this.get('model.provider.reviewsWorkflow').toLowerCase()]
+        }
+        return CLASS_NAMES[this.get('model.reviewsState')];
+    }),
+
+    // TODO: move to 'content/index.hbs' once null value is updated
+    attributeBindings: ['hidden'],
+    hidden: Ember.computed('model.provider.reviewsWorkflow', function() {
+        return this.get('model.provider.reviewsWorkflow') === 'none';
+    }),
+
+    reviewerComment: Ember.computed('firstReviewLog', function () {
```

Comment:
Either `{{firstReviewLog.comment}}` or `reviewerComment: Ember.computed.alias('firstReviewLog.comment')`, if you'd rather.